securedrop/source_templates/base.html

```diff
@@ -26,6 +26,9 @@
       {% endblock %}
 
       <div class="panel selected">
+	{% if 'logged_in' in session %}
+	<a href="{{ url_for('logout') }}" class="btn pull-right">Log Out</a>
+	{% endif %}
```

Comment:
Ok, then this is fine for now. We can come back to it later if we decide to rework the design for the Source Interface in a more holistic manner.